content/wasm-languages/javascript.md

```diff
@@ -26,6 +25,80 @@ There are three popular ways of building JavaScript into WebAssembly.
 
 Recently, Suborbital has introduced a version of [Javy](https://github.com/suborbital/javy) that supports some of their extensions.
 
+An experimental SDK for building Spin apps using JavaScript (and TypeScript) was [introduced in a Fermyon blog article](https://www.fermyon.com/blog/spin-js-sdk) back in December 2022. This [Spin SDK for Javascript](https://github.com/fermyon/spin-js-sdk) borrows heavily from Javy, using the same approach of providing a CLI utility to convert a JS file into a Wasm file.
+
+## Usage
+
+The Spin SDK makes it very easy to build Javascript/TypeScript Wasm applications simply by using a Spin template that handles all of the heavy lifting. If you would like to try out the Spin SDK for Javascript please follow along with the example below.
+
+## Example
+
+In this example, we will create a Spin application using Javascript.
```

Comment:
This paragraph seems to recapitulate the last sentence of the "usage" paragraph?